testcases/master/template-android.jinja2

```diff
@@ -0,0 +1,22 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% set OS_INFO = OS_INFO|default("android") %}
+{% set DEPLOY_OS = DEPLOY_OS|default("android") %}
+
+{# urls #}
+{% set DOCKER_IMAGE = DOCKER_IMAGE|default("yongqinliu/linaro-android-docker:0.1") %}
+{% set use_docker_image_for_test_target = use_docker_image_for_test_target|default(true) %}
+
+{# variables needs to be converted to be used by the root master.jinja2 #}
+{% set LAVA_JOB_PRIORITY = LAVA_JOB_PRIORITY|default("medium") %}
+
+{########## LAVA JOB DEFINITION ##############}
+{# override the job_name block defined in testcases/master/template-master.jinja2 #}
+{# with the BUILD_NUMBER in the middle, it's possible to get the sub job name #}
+{# from this lava job name information #}
+{% block job_name %}{{PROJECT_NAME}}-{{OS_INFO}}-{{KERNEL_BRANCH}}-{{BUILD_NUMBER}}-{{test_name}}{% endblock job_name %}
```

Comment:
This is a great fix that should be fixed in template-master.jinja2 so all projects get this fix.

Reply:
submitted this in a separate PR #200, let's check and merge there.